cobalt/build/gn.py

```diff
@@ -96,7 +96,11 @@ def configure_out_directory(out_directory: str, platform: str, build_type: str,
 
   gn_command = ['gn', 'gen', out_directory] + gn_gen_args
   print(' '.join(gn_command))
-  subprocess.check_call(gn_command)
+  try:
+    subprocess.check_call(gn_command)
```

Comment:
Replace this with `subprocess.check_call(gn_command, check=True)` to get any GN error message.

Reply:
@oxve this causes android builds to fail

Reply:
Keep as-is. Doesn't seem worth digging into